research/query_service/ir/v6d_ffi/build.rs

```diff
@@ -56,7 +51,7 @@ fn codegen_inplace() -> Result<(), Box<dyn std::error::Error>> {
 
     if cfg!(target_os = "linux") {
         println!("cargo:rustc-link-lib=mpi_cxx");
-        if Path::new("/usr/ali/alicpp/built/gcc-4.9.2").exists() {
+        if std::path::Path::new("/usr/ali/alicpp/built/gcc-4.9.2").exists() {
```

Comment:
This is very hack~~

Reply:
Can be removed, as alicpp (4.9.2) is no longer our target platform.